### ISSUE TITLE
gksqt Fatal IO error

### ISSUE THREAD

**Issue by paulraines68** [jheinen/GR.jl#351]
Just started with Julia and downloaded 1.5.2 to run on CentOS7.  I ran Julia and added "GR" as in the instructions.  When I try to run the histogram(rand(10000)) example I get this error:

```
julia> using GR
julia> histogram(rand(10000))
julia> Gdk-Message: 11:13:36.533: gksqt: Fatal IO error 2 (No such file or directory) on X server :1001.

```

 which seems better explained once I tried it by running gksqt manually in another window first:

```
$ /homes/11/raines/.julia/packages/GR/BwGt2/deps/gr/bin/gksqt
GKS: file open error (/usr/local/gr/fonts/gksfont.dat)
open: No such file or directory
Gdk-Message: 10:23:57.578: gksqt: Fatal IO error 2 (No such file or directory) on X server :1001.
 
```
I see Julia installed GR into my ~/.julia directory along with a bunch of other stuff.  The fonts are there:

```
$ find /homes/11/raines/.julia/ -name gksfont.dat
/homes/11/raines/.julia/packages/GR/BwGt2/deps/gr/fonts/gksfont.dat

```

So why is it looking in this non-existant /usr/local/gr?

Anyway, after copying ~/.julia/packages/GR/BwGt2/deps/gr to /usr/local/gr the font error goes away but the histogram still fails with the same Fatal IO Error.  In case it is relevent, I am connected to the CentOS7 machine over VNC working from home due to the pandemic.

Also, how do you install extra Packages globally?  Not in ~/.julia but in the same place as the main Julia 1.5.2 install for all users of the system?

**Comment by jheinen** [comment]
I could not reproduce this behaviour on a fresh CentOS installation. Do you probably have older run-times installed in `/homes/11/raines/.julia/packages/GR/`? If so, could you please remove those directories and `] update`.

If this doesn't help, please try `ENV["GRDIR"]=""; ] build GR`.

With the next release (see latest commits master branch), such problems should be fixed.

**Comment by paulraines68** [comment]
That did not help.  But this definitely appears to be an issue with NoMachine (sorry, I said VNC before but I was actually using NoMachine as the remote desktop method).    I tried running in the NoMachine session using VirtualGL and that worked fine.  I also did run VNC and try that and it worked too.  This is likely some strangeness between Qt's use of the X11 libs and what NoMachine supports/does and nothing specific to GL.jl so I will close this.
